### PR TITLE
add media query in bookDetail.html

### PR DESCRIPTION
**Pull request by Gianni-Nicolas**
**Summary**
Add media query in bookDetail.html

**Screenshot**

Captura de Pantalla 2020-03-09 a la(s) 17 30 48

**Trello**

https://trello.com/c/T5wgUHlZ/34-responsive

### CODE REVIEW

File: scss/detail.scss [hunk 1]
```diff
@@ -111,3 +115,83 @@ $container-book-elements-height: 368px;
...
+    height: 667px;
+    width: 376px;
+  }
+  
+  .container-button {
+    flex-direction: column-reverse;
+  }
+
+  .go-to-index {
+    display: flex;
```
Comment (martinmona) [inline scss/detail.scss]: :thinking:

File: scss/detail.scss [hunk 2]
```diff
@@ -111,3 +115,83 @@ $container-book-elements-height: 368px;
...
+    width: 212px;
+    margin-right: 0;
+  }
+
+  .book-information {
+    margin-top: 20px;
+  }
+
+  .title {
+    height: 39px;
```
Comment (martinmona) [inline scss/detail.scss:297]: don't set height for text, you have line-height already

File: scss/detail.scss [hunk 3]
```diff
@@ -111,3 +115,83 @@ $container-book-elements-height: 368px;
...
+    width: 300px;
+    @include font-config(16px, 700);
+    line-height: 38px;
+  }
+
+  .title-description {
+    height: 16px;
```
Comment (martinmona) [inline scss/detail.scss:311]: same

File: scss/detail.scss [hunk 4]
```diff
@@ -111,3 +115,83 @@ $container-book-elements-height: 368px;
...
+    width: 98px;
+    @include font-config(16px, 700);
+    line-height: 19px;
+  }
+
+  .detail {
+    height: 22px;
```
Comment (martinmona) [inline scss/detail.scss]: same

File: scss/detail.scss [hunk 5]
```diff
@@ -111,3 +115,83 @@ $container-book-elements-height: 368px;
...
+    @include font-config(14px, 700);
+    line-height: 24px;
+    margin-bottom: 20px;
+    width: 300px;
+  }
+  
+  .description {
+    font-size: 14px;
+    line-height: 24px;
+    height: 22px;
```
Comment (martinmona) [inline scss/detail.scss]: image

**Review by martinmona — CHANGES_REQUESTED** [review]

**Review by martinmona — APPROVED** [review]

**Review by nvescio — APPROVED** [review]

**Pull request merged**